tests/lapi/debug_getinfo_test.lua

```diff
@@ -0,0 +1,92 @@
+--[[
+SPDX-License-Identifier: ISC
+Copyright (c) 2023-2025, Sergey Bronnikov.
+
+Parameter 'what' of 'debug.getinfo' cannot start with '>',
+https://www.lua.org/bugs.html#5.4.2-2
+
+Access to debug information in line hook of stripped function,
+https://github.com/lua/lua/commit/ae5b5ba529753c7a653901ffc29b5ea24c3fdf3a
+
+Return hook may not see correct values for active local variables when function returns,
+https://www.lua.org/bugs.html#5.3.0-4
+
+The PC out-of-range in lj_debug_frameline(),
+https://github.com/LuaJIT/LuaJIT/issues/1369
+
+LuaJIT segfault in debug.getinfo(),
+https://github.com/LuaJIT/LuaJIT/issues/509
+
+Synopsis: debug.getinfo([thread,] function [, what])
+]]
+
+local luzer = require("luzer")
+local test_lib = require("lib")
+
+local what = {
+    "n", -- fills in the field name and namewhat.
+    "S", -- fills in the fields source, short_src, linedefined,
+         -- lastlinedefined, and what.
+    "l", -- fills in the field currentline.
+    "u", -- fills in the field nups.
+    "f", -- pushes onto the stack the function that is running at
+         -- the given level.
+    "L", -- pushes onto the stack a table whose indices are the
+         -- numbers of the lines that are valid on the function.
+}
```

Comment:
Also, since Lua 5.4 there is another one control character `r`, see [`lua_getinfo()` section](https://www.lua.org/manual/5.4/manual.html#lua_getinfo).

Reply:
Added as weell.